common/flatpak-dir.c

```diff
@@ -7969,9 +8022,10 @@ flatpak_dir_find_local_ref (FlatpakDir   *self,
   if (!ostree_repo_list_refs (self->repo, NULL, &local_refs, cancellable, error))
     return NULL;
 
+  collection_id = flatpak_dir_get_remote_collection_id (self, remote);
   local_ref = find_ref_for_refs_set (local_refs, name, opt_branch,
```

Comment:
I think `local_refs` here is the wrong kind of hash table. (Found by looking at all the callers of `find_ref_for_refs_set()`.)

Reply:
Oops, sorry about that. Completely missed it...